SQL/0000-00-00-schema.sql

```diff
@@ -2286,6 +2287,16 @@ CREATE TABLE subproject (
 INSERT INTO subproject (SubprojectID, title, useEDC, WindowDifference) VALUES (1, 'Control', false, 'optimal');
 INSERT INTO subproject (SubprojectID, title, useEDC, WindowDifference) VALUES (2, 'Experimental', false, 'optimal');
 
+DROP TABLE IF EXISTS `Project`;
+CREATE TABLE `Project` (
+    `ProjectID` INT(2) Default NULL,
+    `Name` VARCHAR(255) NULL,
+    `recruitmentTarget` INT(6) Default NULL,
+    PRIMARY KEY (`ProjectID`)
+)ENGINE = InnoDB  DEFAULT CHARSET=utf8;
+INSERT INTO Project (ProjectID, Name, recruitmentTarget) VALUES (1,'Sample Project' , 50);
+INSERT INTO Project (ProjectID, Name, recruitmentTarget) VALUES (2,'Another Sample Project' , 50);
```

Comment:
From what I understand, since the table is empty at the start, the test expects the array to be empty when it does the assertion. I'm reading the file of the test and there are some lines that I don't understand. 
$API = new \Loris\API\Projects("GET");